src/ui/public/filter_bar/filter_pill/filter_pill.js

```diff
@@ -51,6 +51,20 @@ module.directive('filterPill', function () {
         return _.has(this.filter, 'meta.controlledBy');
       };
 
+      $scope.enableFilterAriaLabel = i18n('common.ui.filterBar.filterPill.enableFilterAriaLabel', { defaultMessage: 'Enable filter' });
```

Comment:
Please don't use `$scope` if we haven't beforehand. just assign this to something lie `this.i18n = { ... }`, which you can access via `pill.i18n` in the controller.